outOfSyncNoti.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
-# A simple shell script to check if node is out of sync with ethereum testnet
-# If node is out of sync, user will be nptofied by OpsGenie
-# Using crontab command: `*/10 * * * * /home/someuser/tmp/` to run this script every 10 mins.
+# A simple shell script to check if node is out of sync with ethereum
+# If node is out of sync, user will be notified by OpsGenie
+# Using crontab command: `*/10 * * * * /home/foo/outOfSyncNoti.sh 'rinkeby' 'opsgeniekeyhere' >> /home/foo/nodeSync.log 2>&1` to run this script every 10 mins.
```

Comment:
port param seems to be missing in the command
also let's move the command to the new line for better readability